server/models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  availability: {
+    type: [Date],
+  },
+  photo: {
+    type: String,
+  }
+});
+
+module.exports = User = mongoose.model("profile", profileSchema);
```

Comment:
```suggestion
module.exports = User = mongoose.model("Profile", profileSchema);
```
Optional: Please resort to using PascalCase for model names :)

Reply:
The provided user model had it this way so I did the same :P Changed now, good to merge?